plugins/modules/zos_copy.py

```diff
@@ -563,6 +563,61 @@
     returned: success
     type: str
     sample: SAMPLE.SEQ.DATA.SET
+dest_created:
+    description: Boolean value that indicate the creation of dest.
```

Comment:
I think you could rework this to "Indicates whether the module created the destination".